packages/e2e-test-app/tsconfig.json

```diff
@@ -1,8 +1,7 @@
 {
   "extends": "@rnw-scripts/ts-config",
   "compilerOptions": {
-    "esModuleInterop": true,
-    "types": ["jest", "webdriverio"]
```

Comment:
This is a leftover, from when we used `webdriverio` globals directly. We now instead use it indirectly through `@react-native-windows/automation`, unless there are usages I'm missing.